docs/operations/auth-ldap.md

```diff
@@ -254,7 +254,7 @@ Before you start to set up LDAPS in Druid, you must [configure Druid for LDAP au
 
 Complete the following steps to set up LDAPS for Druid. See [Configuration reference](../configuration/index.md) for the location of the configuration files. 
 
-1. Import the CA certificate for your LDAP server or a self-signed certificate into the truststore location saved as `druid.client.https.trustStorePath` in your `common.runtime.properties` file.
+1. Either create a new java trust store and import the CA certificate (or self-signed certificate) for your LDAP server into the newly created ldap truststore or import the CA certificate (or self-signed certificate) into the truststore specified by the `druid.client.https.trustStorePath`  property located in your `common.runtime.properties` file. The example below ilustrates the option with one key store for both https clients and ldap clients but you could use a separate dedicated trust store just for ldap if you wish. 
```

Comment:
A minor rewording and formatting change so that it renders more nicely: 
![image](https://github.com/apache/druid/assets/53799971/699636d7-6b02-4505-8253-3e3715c7e2ab)


```suggestion
1. Import the CA or self-signed certificate for your LDAP server into either a newly created LDAP trust store or the trust store specified by the `druid.client.https.trustStorePath`  property located in your `common.runtime.properties` file.

   The example below illustrates the option with one key store for both HTTPS clients and LDAP clients, but you can use a separate dedicated trust store just for ldap if you wish. 
```